package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "jest",
     "babel": "./node_modules/.bin/webpack -d --watch",
```

Comment:
Aren't we transpiling with webpack here? The babel script name might be somewhat confusing for others reading your code I would think because this just runs webpack.